Jenkinsfile.treecompose

```diff
@@ -50,6 +50,7 @@ node(env.NODE) {
         stage("Compose Tree") {
             sh "rpm-ostree compose tree --repo=${repo} ${manifest} --write-commitid-to $WORKSPACE/commit.txt"
             currentBuild.description = '🆕 commit ' + readFile('commit.txt');
+            sh 'rev=$(cat commit.txt) && rpm-ostree db diff ${rev}^ ${rev}'
```

Comment:
Would `$WORKSPACE/commit.txt` be better here, since we've been bitten in the past when we didn't use that?

Reply:
I think the issues with needing `$WORKSPACE` elsewhere is because of `cd`; in the rdgo code we do:
`            sh "cd ${rdgo} && rpmdistro-gitoverlay build --touch-if-changed $WORKSPACE/rdgo.stamp --logdir=log"`

But we're not doing that here.